bin/init.sh

```diff
@@ -89,6 +91,15 @@ if [ ! -f ${ISTIO_OUT}/envoy ] ; then
     cp $OUT/envoy-$PROXYVERSION ${ISTIO_OUT}/envoy
 fi
 
+if [ ! -f /usr/local/bin/helm -a ! -f ${ISTIO_OUT}/helm ] ; then
+    # Install helm. Please keep it in sync with .circleci
+    cd /tmp && \
+        curl -Lo /tmp/helm.tgz https://storage.googleapis.com/kubernetes-helm/helm-${HELM_VER}-linux-amd64.tar.gz && \
```

Comment:
can we make this optional ? not everyone should have to install helm ? (only if you touch the templates,...)

not every build (lint, unit test, etc...) need helm either

Reply:
We plan to use helm to generate istio.yaml and test configs - so it won't be optional. 
Also helm ( full helm, with tiller ) will need to be part of one of the integration tests.

Reply:
this is a bit risky though because helm can easily become required accidentally that way
it's also not hermetic, we should avoid downloading stuff
maybe the helm client can be vendored ? it should already have most of it as we use the libs in gen-deploy ?

in short I'm concerned it's really easy after that to break the "istio-auth.yaml" is all you need to install a basic istio (demo) setup - and pass all the tests

Reply:
it still shouldn't be required